src/AppBundle/Controller/AdminScoreController.php

```diff
@@ -105,8 +105,10 @@ public function resultAction($project, $username)
                 AND log_namespace=0
                 AND log_deleted=0 and log_user_text=:username
         UNION
-        SELECT 'blocks' as source, COUNT(*) as value FROM $loggingTable
-            WHERE log_type='block' AND log_action='block' AND log_namespace=2 AND log_deleted=0 AND log_title=:username
+        SELECT 'blocks' as source, COUNT(*) as value FROM $loggingTable l
+            INNER JOIN $userTable u ON l.log_user = u.user_id
+            WHERE l.log_type='block' AND l.log_action='block'
+            AND l.log_namespace=2 AND l.log_deleted=0 AND u.user_name=:username
```

Comment:
Okay, I'm convinced.  Please fix my other comment and we'll give this the thumbs up.  

To clarify my other comment - in order to access a table you use the [MediaWiki table](https://upload.wikimedia.org/wikipedia/commons/9/94/MediaWiki_1.28.0_database_schema.svg) rather than the labs table.  We then map it using app/config/table_map.yml